csharp/Bot/HelloWorldTrigger.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Interfaces;
+using Octokit;
+using Storage.Local;
+using Storage.Remote.GitHub;
+using System.Collections.Generic;
+
+namespace csharp
+{
+    class HelloWorldTrigger : ITrigger<Issue>
+    {
+        private readonly GitHubStorage gitHubAPI;
+
+        private readonly List<IFile> files;
+
+        public HelloWorldTrigger(GitHubStorage gitHubAPI,FileStorage fileStorage, string fileSetName)
+        {
+            this.gitHubAPI = gitHubAPI;
+            files = fileStorage.GetFilesFromSet(fileSetName);
+        }
+
+        public void Action(Issue obj)
+        {
+            foreach (var file in files)

```

Comment:
You can use `fileStorage.GetFilesFromSet(fileSetName)` directly here, it is pretty fast. No need `files` field, use `fileStorage` and `fileSetName` fields instead.